backend/share-handler.js

```diff
@@ -23,7 +24,8 @@ async function handler (event, context) {
   const key = `shares/${id[0]}/${id[1]}/${id}`
 
   const filename = event?.queryStringParameters?.filename
-  const contentDisposition = `attachment; filename="${filename}"`
+  const sanitizedFilename = filename && sanitizeFilename(filename)
```

Comment:
Nice! I really like this solution. Would it be worthwhile adding a comment here to describe the kind of problem we are protecting against? I would be interested to read more, so maybe a link to some external resource can help here.